spec/app/ics-028-cross-chain-validation/methods.md

```diff
@@ -701,16 +827,24 @@ function InitGenesis(gs: ConsumerGenesisState): [ValidatorUpdate] {
   - The consumer CCV module is in the initial state.  
 - **Postcondition**
   - The capability for the port `ConsumerPortId` is claimed.
-  - A client of the provider chain is created and the client ID is stored into `providerClient`.
-  - `consumerUnbondingPeriod` is set to `ComputeConsumerUnbondingPeriod(gs.providerClientState.unbondingTime)`.
+  - `preCCV` is set to `gs.preCCV`.
+  - If `preCCV == true`, the ID of the client on which the connection with `gs.connId` is built is stored into `providerClient`.
+  - Otherwise, a client of the provider chain is created and the client ID is stored into `providerClient`.
+  - `consumerUnbondingPeriod` is set using the unbonding period from the client of the provider chain.
   - `HtoVSC` for the current block is set to `0`.
-  - The `validatorSet` mapping is populated with the initial validator set.
+  - The `ccvValidatorSet` mapping is populated with the initial validator set.
+  - The ID of the distribution token transfer channel is set to `gs.distributionChannelId`.
```

Comment:
if defined

Reply:
If it's not define, it will be set to `""` (the default value of `gs.distributionChannelId`)  and then a distribution channel will be created in `onChanOpenAck`.